### ISSUE TITLE
Game no response when right click Lava Pump, Pressure Pump, Electric Pump and Redstone Pump

### ISSUE THREAD

**Issue by bakaxyf** [K-4U/Hydraulicraft#155]

**Comment by K-4U** [comment]
This is weird though. Do you have any other mods that might be interfering? I have never heard of this happening before.

Are you sure there are no stacktraces in your log?


Would like some confirmation from other people.

**Comment by bakaxyf** [comment]
https://gist.github.com/bakaxyf/a397b89bc9426dad45af

**Comment by K-4U** [comment]
I cannot find anything in your log that would prove any difficulty in opening the GUI.. Are other GUI's working?

**Comment by bakaxyf** [comment]
Yes, works fine.

**Comment by K-4U** [comment]
I'm still confused by this. I've gotten info that it may be because of the localization. Are you running english localization?

**Comment by bakaxyf** [comment]
You are right. Because of the localization. They work fine in running English localization but no response in Chinese localization

**Comment by K-4U** [comment]
Is this with the modified localisation that you PR'd?

**Comment by bakaxyf** [comment]
Nope. Used [HydCraft-1.7.10-2.1.239-universal] (http://jenkins.k-4u.nl/job/Hydraulicraft/239/) unmodified.

**Comment by K-4U** [comment]
We've managed to trace it down to the font renderer not being able to run the chinese localization.

As of yet, unsure of how to best fix. Will research some more.

**Comment by alalleo** [comment]
[23:35:19] [Client thread/INFO] [inventorytweaks]: k4unl.minecraft.Hydraulicraft.client.GUI.GuiPump
[23:35:19] [Client thread/INFO] [inventorytweaks]: k4unl.minecraft.Hydraulicraft.containers.ContainerPump
And the Minecraft freezes.
Russian localization.
I am use modpack "Modular Mayhem" http://forum.feed-the-beast.com/threads/1-7-10-modular-mayhem.120584/

**Comment by CallMeFoxie** [comment]
If Russian localization uses Cyrilic it might be the same issue then.

**Comment by alalleo** [comment]
Yes. Cyrilic.
